README.md

```diff
@@ -0,0 +1,31 @@
+# matrix-auth-plugin
+Matrix-based authorization strategies for Jenkins
+
+Sample pipeline (assumes matrix based security enabled on jenkins):
+
+    properties ([
+  		  [$class: 'AuthorizationMatrixProperty',
```

Comment:
Yeah, I'd block on that - I think that's critical. Add the `structs` dependency and add `@Symbol('authorizationMatrix')` to the descriptor in `AuthorizationMatrixProperty` and you're in good shape.